themes/SuiteP/css/suitep-base/messages.scss

```diff
@@ -0,0 +1,28 @@
+/**** Messages ***/
+
+.message {
```

Comment:
you don't have to add specific stylesheet for messages as it is already pre-defined by bootstrap. Can you use the bootstrap css selection, please.
https://getbootstrap.com/docs/4.0/components/alerts/

Reply:
Great! 
Thanks @gymad